### PR TITLE
Issue #29 Adding travis CI and make test

### LINKED ISSUE #29: Add CI for make, make cross and make test
We need to add e.g. travis for test our PRs for  make, make cross

  * [x] Add `Makefile`
  * [x] Travis integration

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Fixes #29 

Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

**Comment by anjannath** [comment]
@LalatenduMohanty This also needs rebase!

File: pkg/crc/logging/logging.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/code-ready/crc/pkg/crc/output"
 )
 
-func Log(s string, args ... interface{}) {
+func Log(s string, args ...interface{}) {
```
Comment (gbraad) [inline pkg/crc/logging/logging.go]: please rebase

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+language: go
+
+go:
+- 1.11.4
+
+install:
+- echo "Don't run anything."
```
Comment (gbraad) [inline .travis.yml]: is the install section a necessity ?

**Comment by praveenkumar** [comment]
Closing this now, updated to #71

**Pull request closed without merging**